### PR TITLE
feat(VStepper): Flat prop

### LINKED ISSUE #4076: [Feature Request] Add flat property to stepper
### Problem to solve
Remove box-shadow same as with v-card flat

### Proposed solution
flat property as in v-card

### PR DESCRIPTION
**Pull request by ElijahKotyluk**
## Description

resolves #4076 
## How Has This Been Tested?

Jest and Playground.

## Markup:

<details>

```vue
<template>
  <div>
    <v-stepper flat>
      <v-stepper-header>
        <v-stepper-step step="1">
          Step 1
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="2">
          Step 2
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="3">
          Step 3
        </v-stepper-step>
      </v-stepper-header>
    </v-stepper>

    <v-stepper
      class="mt-12"
      elevation="4"
    >
      <v-stepper-header>
        <v-stepper-step step="1">
          Step 1
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="2">
          Step 2
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="3">
          Step 3
        </v-stepper-step>
      </v-stepper-header>
    </v-stepper>

    <v-stepper
      class="mt-12"
      elevation="24"
    >
      <v-stepper-header>
        <v-stepper-step step="1">
          Step 1
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="2">
          Step 2
        </v-stepper-step>

        <v-divider></v-divider>

        <v-stepper-step step="3">
          Step 3
        </v-stepper-step>
      </v-stepper-header>
    </v-stepper>
  </div>
</template>
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by jacekkarczmarczyk** [comment]
I think `elevation` prop should be added as well to be in line with other components

**Comment by Mert75** [comment]
Looks good to me, especially with the `elevation` prop added.

File: packages/vuetify/src/components/VStepper/VStepper.sass [hunk 1]
```diff
@@ -174,6 +174,9 @@
     .v-stepper__content, .v-stepper__wrapper
       transition: .3s map-get($transition, 'swing')
 
+  &--flat
+    +elevation(0, true)
```
Comment (KaelWD) [inline packages/vuetify/src/components/VStepper/VStepper.sass:178]: Why does this need to be `!important`?

Comment (ElijahKotyluk) [reply packages/vuetify/src/components/VStepper/VStepper.sass:178]: It doesn't, I did a dummy.

**Comment by johnleider** [comment]
I decided to add the entire VSheet functionality and resolved this in ae51c0760

Sorry for any wasted time.

**Pull request closed without merging**